magda-web-client/src/Components/SearchFacets/FacetRegion.js

```diff
@@ -122,6 +123,19 @@ class FacetRegion extends Component {
         );
     }
 
+    renderNoOption() {
```

Comment:
Can we pull this out into a separate little component seeing how we're using it in two places?